### DISCUSSION TITLE
a-gps data downloaded while in airplane mode

### DISCUSSION THREAD

**Discussion by norayr** [osmandapp/OsmAnd#11952] [General]
i have the phone in airplane mode here, but gps and wifi are enabled. 
i hope that means gsm modem is off (or in sleep mode). i actually don't even have a sim card in the phone. 

when i press the location button (second from right, circle with a dot inside) i get this message: 

A-GPS data downloaded. 

i wonder what is happening here. does my phone see cellurar towers around and requests information about those from internet servers, or the application just doesn't handle this case correctly and actually does not see any towers, and may be didn't even request the data. 

thank you.

**Comment by vshcherb** [comment]
I guess the app message is not correct ;-)

**Comment by styrheim** [comment]
Sounds OK to me. https://en.wikipedia.org/wiki/Assisted_GNSS says A-GPS is cached info about the satellites, info that may take long time to download, if downloaded from the satellites themselves. there does not seem to be any cellular towers involved.

(But – _listening_ for cellular towers would be allowed in flight mode?)
